rskj-core/src/main/java/co/rsk/rpc/netty/Web3WebSocketServer.java

```diff
@@ -83,7 +83,7 @@ protected void initChannel(SocketChannel ch) throws Exception {
                     p.addLast(new HttpServerCodec());
                     p.addLast(new HttpObjectAggregator(maxAggregatedFrameSize));
                     p.addLast(new WriteTimeoutHandler(serverWriteTimeoutSeconds, TimeUnit.SECONDS));
-                    p.addLast(new RskWebSocketServerProtocolHandler("/websocket", maxFrameSize));
+                    p.addLast(new RskWebSocketServerProtocolHandler("/", maxFrameSize));
```

Comment:
I understand this allows any path, doesn't it? Shouldn't we restrict it to only accept `ws://your-ip:4445/websocket` and `ws://your-ip:4445`?

Reply:
I see. Any way, looks like we support any path for http endpoint too.

Btw.. please check the failed pipeline